internal/doc/draft-user-guide-v1.adoc

```diff
@@ -718,11 +721,96 @@ When `expose` is **not** set to `true`, the Knative service is labeled with `ser
 
 To configure secure HTTPS connections for your Knative deployment, see link:++https://knative.dev/docs/serving/using-a-tls-cert/++[Configuring HTTPS with TLS certificates] for more information.
 
+=== Network Policies
+
+Network policies for an application are created by default to isolate ingress traffic. To disable creating network
+policies for an application, set `.spec.networkPolicy.disable` to `true`:
+
+[source,yaml]
+----
+apiVersion: rc.app.stacks/v1beta2
+kind: RuntimeComponent
+metadata:
+  name: my-app
+spec:
+  networkPolicy:
+    disable: true
+----
+
+==== Network Policy Prerequisites
+
+OpenShift provides support for network policies by default. Users deploying to other Kubernetes platforms will need to
+ensure that their network plugin supports network policies as described
+link:https://kubernetes.io/docs/concepts/services-networking/network-policies/#prerequisites[here].
+
+==== Default Network Polices
+
+* The default network policy created for applications that are not exposed limits ingress traffic to pods in the same
+namespace that are part of the same application. Traffic is limited to only the ports configured by the service.
+* For exposed applications on OpenShift, the network policy limits ingress traffic from the OpenShift ingress
+controller on the ports configured in the service.
+* For exposed applications on other Kubernetes platforms, ingress traffic is allowed from any pods in any namespace on the
+ports configured in the service.
+* On OpenShift, ingress traffic is also allowed from the OpenShift monitoring stack.
+
+==== Configuring the Network Policy
+
+The network policy can be configured to allow either certain namespaces and/or pods using
+`.spec.networkPolicy.namespaceLabels` and `.spec.networkPolicy.fromLabels`, respectively.
+
+By default, `.spec.networkPolicy.namespaceLabels` is set to the same namespace that the application is deployed to, and
+`.spec.networkPolicy.fromLabels` is set to select pods that belong to the same application specified by
+`.spec.applicationName`.
+
+Example of allowing ingress traffic from pods in the same namespace labeled with `role: frontend`:
```

Comment:
"from pods labeled with `role: frontend` and are from the same namespace" - so users don't mistakenly think it's referring to namespace labels